build/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM golang:1.11
+
+RUN apt-get update && apt-get install -y make git apt-transport-https ca-certificates curl software-properties-common build-essential zip wget xmlsec1
```

Comment:
What docker image is this? It would be nice if there was some way to tell. (maybe put it in a sub directory with the image name? Or separate repo?

Reply:
added